packages/labs/compiler/src/lib/template-transform.ts

```diff
@@ -283,6 +283,13 @@ class CompiledTemplatePass {
       'pre:node': (node): boolean | void => {
         if (isElementNode(node)) {
           const attributesToRemove = new Set<unknown>();
+          if (node.tagName.includes(marker)) {
+            // Do not compile template if a marker was inserted in tag name
+            // position.
+            // TODO: Provide a diagnostic here.
+            shouldCompile = false;
+            return true;
```

Comment:
Is there value to returning `true` here? There's no need to traverse into the children if we're not compiling, right?

Reply:
Whoops! This should be false.